tests/scripts/generate_bignum_tests.py

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/env python3
+"""Generate test data for bignum functions.
+
+With no arguments, generate all test data. With non-option arguments,
+generate only the specified files.
+
+Class structure:
+
```

Comment:
This is not enforced but it could make sense to implement docstings in files after the license using the `__doc__ = """ ...` format.

Reply:
Why?

In any case the file description (docstring in Python) goes before the license (like in [`*.h`](https://mbed-tls.readthedocs.io/en/latest/kb/development/mbedtls-coding-standards/#header-files) and [`*.c`](https://mbed-tls.readthedocs.io/en/latest/kb/development/mbedtls-coding-standards/#source-files)).

Reply:
The idea behind docstrings was to have accessible manuals in the even during the interpreter execution runtime. e.g `print(__doc__)` . 

Because those file attributes are set on the beggining of the file, it is common to use intermediate variables when including large licence or docstrings

e.g 

~~~~~
"""random_file.py: Some dosctring describing it"""

__license__ = "GPLv2"
__version__ = "1.0"
__author__  = "A Developer"
__email__   = "developer@mail.com"
~~~~~

Or

~~~~~
__doc__     = _dc_
__license__ = "GPLv2"
__version__ = "1.0"
__author__  = "A Developer"
__email__   = "developer@mail.com"

_dc_ = """ random_file.py: Some multiline
                 dosctring describing it 
           """
~~~~~

Anyhow we do not use that format, and it is extremely unlikely to scale our scripts to the size that it would be required. 

Could it be worth to mention in our guidelines ?